<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="clifford">
  <xacro:include filename="clifford_core.xacro"/>
  <xacro:include filename="joint_group_position_controller.xacro"/>
</robot>